src/joint_types.jl

```diff
@@ -14,7 +14,10 @@ function _local_coordinates!(jt::JointType,
 end
 
 function _global_coordinates!(jt::JointType, q::AbstractVector, q0::AbstractVector, ϕ::AbstractVector)
-    q .= q0 .+ ϕ # TODO: allocates on 0.5
```

Comment:
BTW, the StaticArrays readme mentions that you can apparently do:

```julia
x .= (+).(x, pos)
```

to get fusing broadcast of + and = on julia 0.5.